mkdocs.yml

```diff
@@ -41,6 +41,7 @@ nav:
     - Karpenter: 'addons/karpenter.md'
     - Keptn: 'https://github.com/keptn-sandbox/keptn-eks-blueprints-addon'
     - Knative: 'addons/knative-operator.md'
+    - Komodor: 'https://github.com/komodorio/komodor-eks-blueprints-addon'
```

Comment:
As mentioned in the patterns PR [here](https://github.com/aws-samples/cdk-eks-blueprints-patterns/pull/58#pullrequestreview-1283179852), once you are able to implement secrets for addons, please update your official documentation with an example of how to use the addon with secret vs API key as a property.